vnext/Shared/Networking/IRedirectEventSource.idl

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.ReactNative {
+
+  /// <summary>
+  /// An internal API to extend the phases of HTTP redirection.
+  /// See RedirectHttpFilter.
+  /// This interface may add other methods reflecting later redirection stages
+  /// </summary>
```

Comment:
note that these comments don't end up anywhere.
Since they are in the IDL, these types will be exposed in the WinMD (there's no "private"), and they need the DOC_STRING attributes in order for us to generate docs for them

Reply:
As discussed offline, will drop the IDL file (this is not a public interface) and declare it as a C++ internal type.

Reply:
The IDL file has been dropped in favor of an internal interface.